src/main/resources/db/migration/V2_0__modify_appeal.sql

```diff
@@ -4,3 +4,28 @@ alter table attachment drop foreign key FK_appeal_id;
 alter table attachment drop column appeal_id;
 drop table appeal_extra_parameters;
 drop table appeal;
+
+create table appeal (
+    version integer,
+    appeal_concern_communicated_at datetime(6),
+    created datetime(6),
+    decision_id bigint,
+    errand_id bigint,
+    id bigint not null auto_increment,
+    registered_at datetime(6),
+    updated datetime(6),
+    description text,
+    status enum ('NEW','REJECTED','SENT_FOR_TRIAL','COMPLETED'),
+    trail_status enum ('NOT_CONDUCTED','NOT_RELEVANT','APPROVED','REJECTED'),
```

Comment:
There has been some back and forth regarding the enum datatype in the database and I think we settled on avoiding it. Instead having a varchar, so additions or removals of enum values doesn't affect the datamodel.